tests/test_fluent_fixes.py

```diff
@@ -31,3 +31,38 @@ def test_1364(new_solver_session):
     assert report_def.zone_names.allowed_values() == ["fluid"]
 
     assert report_def.expr_list.allowed_values() == None
+
+
+def test_637_974_1744(new_solver_session):
```

Comment:
I think it is indeed a great idea to indicate the GitHub issue numbers for the tests. But can we use a more descriptive test name? I think it would be better if the test name described what the test is for, and then we can add, as a code comment, or at the end of the test name, what the associated GitHub issue numbers are as additional information. 

If you could change `test_1364` in this same file to a more descriptive name as well I think that'd be great.

Reply:
Yes, having the IDs at the end of a descriptive name will be OK.

Reply:
Actually it was the requirement at the time of creating this test file to use issue numbers because main purpose is to track fixes that we have provided, I think numbers are perfect and compact for this case.